scripts/lib/net-util.sh

```diff
@@ -20,6 +20,15 @@ wait_for_dev()
     fi
 }
 
+calc_mtu_size()
+{
+   iface=$(ip route show default 0.0.0.0/0|cut -d' ' -f5)
+   iface_mtu=$(ip link show dev eth0 | grep mtu | awk '{print $5}')
+
+   # remove enough for IPSEC and IP headers from network agent.
+   echo $(($iface_mtu-98))
```

Comment:
98<sup>[citation needed]</sup>..  More info on where that number came from would be nice.  e.g. is this also enough for IPv6?